### ISSUE TITLE
Doctrine Configuration Methods

### ISSUE THREAD

**Issue by FabianKoestring** [doctrine/DoctrineORMModule#358]
Is there a way to define a default repository for entities?
I cant find anything in the documentation, but i remember something like this.

```php
return array(
    'doctrine' => array(
        'configuration' => array(
            'orm_default' => array(
                'query_cache' => 'filesystem',
                'result_cache' => 'filesystem',
            ),
            'methods' => array(
                'setMethods' => array(
                    'setDefaultRepositoryClassName' => array(
                        'defaultRepositoryClassName' => 'App\Repository\DefaultRepository'
                    ),
                )
            )
        ),
    ),
);
```

Does this work?

**Comment by bakura10** [comment]
It seems that this is addressed by #264

**Comment by bakura10** [comment]
I've merged #375 that includes this feature. Will likely be include in DoctrineORMModule 0.9 (that will be tagged once Doctrine ORM 2.5 is out).
